pkg/utils/utils_test.go

```diff
@@ -172,6 +172,28 @@ var _ = Describe("In the utils package", func() {
 		),
 	)
 
+	DescribeTable("getting NUMA node of device",
+		func(fs *FakeFilesystem, pciAddr string, expected int) {
+			defer fs.Use()()
+			Expect(GetDevNode(pciAddr)).To(Equal(expected))
+		},
+		Entry("reading the device path fails", &FakeFilesystem{}, "0000:00:00.0", -1),
+		Entry("converting the VFs number to integer fails",
+			&FakeFilesystem{
+				Dirs:  []string{"sys/bus/pci/devices/0000:00:00.1"},
+				Files: map[string][]byte{"sys/bus/pci/devices/0000:00:00.1/numa_node": []byte("invalid content")},
+			},
+			"0000:00:00.1", -1,
+		),
+		Entry("finding positive NUMA node",
+			&FakeFilesystem{
+				Dirs:  []string{"sys/bus/pci/devices/0000:00:00.1"},
+				Files: map[string][]byte{"sys/bus/pci/devices/0000:00:00.1/numa_node": []byte("1")},
+			},
+			"0000:00:00.1", 1,
+		),
+	)
```

Comment:
It'll be good to have a test for reading '-1' from 'numa_node' as it's the common  and very frequent case.

Reply:
updated in most recent version